src/Find.css

```diff
@@ -0,0 +1,5 @@
+.select-player {
+    width: 250px;
+    margin: 1em;
```

Comment:
I'm using npm package react-select so the styling on the dropdown happens in the .js file.